data/themes/darktable.css

```diff
@@ -871,7 +870,16 @@ dialog combobox window
   padding: 0.2em 0;
 }
 
-menu menuitem {
+popover
+{
+  background-color: @tooltip_bg_color;
+  border-radius: 2px;
+  border: 0;
+  padding: 0.4em;
+}
```

Comment:
Please put back popover in line 860. That has been done that way to avoid some glitches in some desktop environment. Just add a specific popover part has you done for only padding specific size. I remember anyway that this padding was good quite recently and I'm quite surprise to see that it has been broken. Anyway, thanks for fixing that.

Reply:
Thanks for your feedback.